### ISSUE TITLE
[EN][Warlock][Pet] Minion's name problems

### ISSUE THREAD

**Issue by ashphoenix92** [Atlantiss/BugTracker#4726]
Every time after a resummon or a summoning new minion, it's name changes anew. As far as known, warlock's demon-minions' name can be changed only via Demon Trainers NPCs for a cost, not by itself because of a re-summon or a summoning.

Source - http://wow.gamepedia.com/Warlock_minion#Patch_changes
(look "Notes" section)

**Comment by karhima** [comment]
That's correct, the name should not change every time the minion is summoned. AFAIK this happens to all minions on Atlantiss.
